README.md

```diff
@@ -1,2 +1,83 @@
-Ansible Collection: amazon.aws
-=================================================
+
+
+# Amazon AWS Collection
+[![Shippable build status](https://api.shippable.com/projects/5e4451b6aa9a61000733064c/badge?branch=master)](https://api.shippable.com/projects/5e4451b6aa9a61000733064c/badge?branch=master)
+<!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/amazon.aws)](https://codecov.io/gh/ansible-collections/amazon.aws)-->
+
+The Ansible Amazon AWS collection includes a variety of Ansible content to help automate the management of AWS instances. This collection is maintained by the Ansible cloud team.
+
+## Included content
+
+Click the ``Content`` button to see the list of content included in this collection.
+
+## Installing this collection
+
+You can install the AWS collection with the Ansible Galaxy CLI:
+
+    ansible-galaxy collection install amazon.aws
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: amazon.aws
+    version: 0.1.1
+```
+## Using this collection
+
+
+You can either call modules by their Fully Qualified Collection Namespace (FQCN), such as `amazon.aws.ec2_instance`, or you can call modules by their short name if you list the `ansible.aws` collection in the playbook's `collections` keyword:
```

Comment:
```suggestion
You can either call modules by their Fully Qualified Collection Namespace (FQCN), such as `amazon.aws.ec2_instance`, or you can call modules by their short name if you list the `amazon.aws` collection in the playbook's `collections` keyword:
```
Sorry, totally missed this one until I started copying it over to the community collection!